test/tests/mortar/continuity-3d-non-conforming/continuity_penalty_sphere_hex20.i

```diff
@@ -0,0 +1,126 @@
+[Mesh]
+  second_order = true
+  [file]
+    type = FileMeshGenerator
+    file = spheres_hex20.e
+  []
+  [secondary]
+    input = file
+    type = LowerDBlockFromSidesetGenerator
+    new_block_id = 11
+    new_block_name = "secondary"
+    sidesets = '101'
+  []
+  [primary]
+    input = secondary
+    type = LowerDBlockFromSidesetGenerator
+    new_block_id = 12
+    new_block_name = "primary"
+    sidesets = '102'
+  []
+[]
+
+[Problem]
+  kernel_coverage_check = false
+[]
```

Comment:
Is this necessary? (I can believe that it is ... I would hope it's unnecessary in the case where you have an LM, but I could believe that we get an error if the LM doesn't exist)

Reply:
I get a catastrophic segmentation fault at the very beginning of the run if I don't include that for the penalty tests.

Reply:
catastrophic segmentation fault? That sounds even worse than a normal segmentation fault 😄 We should probably make an issue for that

Reply:
opened #21416